tests/system/src/test/java/test/com/sun/javafx/animation/AnimationTimerTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same, copyright header should keep 2023, even if it is almost completely changed